syntax = "proto3";

service FileService {
  rpc UploadFile(FileRequest) returns (FileResponse);
  rpc DownloadFile(FileRequest) returns (FileResponse);
}

message FileRequest {
  string filename = 1;
}

message FileResponse {
  string message = 1;
}
